android/jni/Android.mk

```diff
@@ -6,9 +6,10 @@ LOCAL_CFLAGS := -DTHREAD_SUPPORT=1 -pthread									\
 	-DFUZZ_MODE=0 -DPERM_FREE_REALLOC=0 -DDISABLE_CANARY=0 -Werror			\
 	-pedantic -Wno-pointer-arith -Wno-gnu-zero-variadic-macro-arguments		\
 	-Wno-format-pedantic -DMALLOC_HOOK=1 -fvisibility=hidden -std=c11		\
-	-DALLOC_SANITY=0 -DUNINIT_READ_SANITY=0 -DCPU_PIN=0 -DEXPERIMENTAL=0	\
+	-DALLOC_SANITY=0 -DUNINIT_READ_SANITY=0 -DCPU_PIN=0 -DEXPERIMENTAL=0		\
 	-DUAF_PTR_PAGE=0 -DVERIFY_BIT_SLOT_CACHE=0 -DNAMED_MAPPINGS=1 -fPIC		\
 	-shared -DDEBUG=1 -DLEAK_DETECTOR=1 -DMEM_USAGE=1 -DUSE_MLOCK=1			\
+	-DSCHED_GETCPU									\
```

Comment:
Please align the trailing `/`

Reply:
what do you mean ?

Reply:
![image](https://user-images.githubusercontent.com/325724/175128776-d4082bb0-1b51-4992-a5d1-49e7094c45b0.png)

See how the trailing `\` isn't aligned with the others?

Reply:
was confused you displayed the opposite character above ;-)